portfolio/src/main/webapp/css/aboutme.css

```diff
@@ -0,0 +1,65 @@
+html, body {
+    height: 100%;
+    width: 100%;
+}
+
+body{
+    margin: 0;
+}
+
+#container {
+    align-items: center;
+    display: flex;
+    flex-direction: column;
+    font-family: Garamond;
+    height: 100%;
+    width: 100%;
+}
+
+#top-ref {
+    width:100%;
+    height: auto;
```

Comment:
Please order the properties alphabetically

Reply:
Done: Removed since it was not in use.